recipes/parliament/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "parliament" %}
+{% set version = "1.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2359299fb09a495faee2622c03c6d3765660656fbc0a4f197c309dbd7a6e2ba6
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - parliament=parliament.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - nose
+    - pip
+    - python >=3.6
+  run:
+    - boto3
+    - jmespath
+    - json-cfg
+    - python >=3.6
+    - pyyaml
+
+test:
+  imports:
+    - parliament
+    - parliament.community_auditors
+  commands:
+    - pip check
+    - parliament --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/duo-labs/parliament
+  summary: parliament audits your AWS IAM policies
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  description: |
+    parliament is an AWS IAM linting library. It reviews policies looking for problems such as:
+
+    * malformed json
+    * missing required elements
+    * incorrect prefix and action names
+    * incorrect resources or conditions for the actions provided
+    * type mismatches
+    * bad policy patterns
```

Comment:
Is that the correct way to declare a list? The output should be like in here: https://github.com/duo-labs/parliament/blob/main/README.md

Reply:
This should be fine. It will get rendered as plain text in some places but most treat it as markdown.

Reply:
Ok, thanks for the feedback!